tests/baselines/reference/templateLiteralTypes3.js

```diff
@@ -124,11 +124,51 @@ type Schema = { a: { b: { c: number } } };
 declare function chain<F extends keyof Schema>(field: F | `${F}.${F}`): void;
 
 chain("a");
+
+// Repro from #46125
+
+function ff1<T extends string>(x: `foo-${string}`, y: `${string}-bar`, z: `baz-${string}`) {
```

Comment:
Before I forget, can we also get a copy of this test with a signature like
```ts
function ff1<T extends string>(x: `foo-${T}`, y: `${T}-bar`, z: `baz-${T}`) {
```
? I figure it should behave the same, but, y'know, coverage. Maybe it can just be more cases on this signature, since it already has a generic parameter for some reason~

Reply:
Will do.